pgmpy/base/DAG.py

```diff
@@ -1228,6 +1246,22 @@ def to_daft(
                 extra_params = edge_params[(u, v)]
             except KeyError:
                 extra_params = dict()
+
+            # Add edge strength as label if requested
+            if plot_edge_strength:
+                if "strength" in self.edges[(u, v)]:
```

Comment:
@Dhiren-Mhatre Why are these checks repeated? After the first check, the values are guaranteed to exist. Right?